server/scripts/webscraper.py

```diff
@@ -0,0 +1,125 @@
+import re
+import requests
+import urllib.request
+import time
+# from test_data import *
+from bs4 import BeautifulSoup
+
+# Started by Jeremy Lim on 20/07/2019
+
+# TO-DO LIST
+# - Get timetable done
+# - Get GenEd done
+# - Majors
+# - Degrees
+# - Add to database
+
+##### COURSES #####
+
+# To print the info of each course
+def printInfo(code, link, name, cred):
+    print("CODE: " + code) # prints the code of the course
+    print("LINK: " + link) # prints the link of the course
+    print("NAME: " + name) # prints the name of the course
+    print("CREDITS: " + cred) # prints the credits of the course
+
+# To go through each letter's links for courses
+def run_course():
+    alphabet = ['A','B','C','D','E','F','G','H','I','L','M','N','O','P','R','S','T','V','Y','Z']
+
+    for letter in alphabet[0:2]:
+
+        # runs the url for the letter search
+        course_url = 'http://legacy.handbook.unsw.edu.au/vbook2018/brCoursesByAtoZ.jsp?StudyLevel=Undergraduate&descr=' + letter
```

Comment:
hard coded constants at top of code; so perhaps do something like 
```response = requests.get(course_url + letter)``` where course_url is the hardcoded string